src/Microsoft.VisualBasic/src/Microsoft/VisualBasic/FileIO/FileSystem.Unix.vb

```diff
@@ -0,0 +1,30 @@
+﻿' Licensed to the .NET Foundation under one or more agreements.
+' The .NET Foundation licenses this file to you under the MIT license.
+' See the LICENSE file in the project root for more information.
+Option Explicit On
+Option Strict On
+
+Imports System
+
+Namespace Microsoft.VisualBasic.FileIO
+    Partial Public Class FileSystem
+        Private Shared Sub ShellCopyOrMove(operation As CopyOrMove, directory As FileOrDirectory, sourceDirectoryFullPath As String, targetDirectoryFullPath As String, showUI As UIOptionInternal, onUserCancel As UICancelOption)
+            Throw New PlatformNotSupportedException(SR.NoShellCopyOrMove)
+        End Sub
+
+        Private Shared Sub ShellDelete(directoryFullPath As String, showUI As UIOptionInternal, recycle As RecycleOption, onUserCancel As UICancelOption, directory As FileOrDirectory)
+            Throw New PlatformNotSupportedException(SR.NoShellDelete)
+        End Sub
+
+        ''' <summary>
+        ''' Stub to prevent compile error, this will not get called on non Windows OS's
+        ''' This would be implemented on non-Windows in the future
+        ''' </summary>
+        ''' <param name="sourceFileFullPath"></param>
+        ''' <param name="destinationFileFullPath"></param>
```

Comment:
Consider removing the comment. The `Throw New PlatformNotSupportedException()` in the body is sufficient.